assignments/function-conversion.js

```diff
@@ -1,23 +1,19 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
-// let myFunction = function () {};
+let myFunction = () => {};
 
-// let anotherFunction = function (param) {
-//   return param;
-// };
+let anotherFunction = (param1) => {return param1;};
```

Comment:
this doesn't need return, etc. It also works as `let anotherFunction = function(param) => param;`